### PR TITLE
RailsDevs French Version

### PR DESCRIPTION
**Pull request by 5andu**
## Pull request checklist

- [x] My code contains tests covering the code I modified
- [x] I linted and tested the project with `bin/check`
- [x] I added significant changes and product updates to the [changelog](CHANGELOG.md)

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Application < Rails::Application
     config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**", "*.{rb,yml}")]
 
     # Permitted locales available for the application
-    config.i18n.available_locales = [:en, :es, :"zh-TW", :"pt-BR"]
```
Comment (joemasilotti) [inline config/application.rb:44]: `fr:` needs to be `:fr` here.

File: app/components/time_component.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <%= time_tag time do %>
-  <%= time_ago_in_words time %> ago
```
Comment (joemasilotti) [inline app/components/time_component.html.erb:2]: "ago" still needs to appear in the UI.

**Review by joemasilotti — CHANGES_REQUESTED** [review]
Thanks for this! I requested two changes and then we should be good to merge.

File: app/components/time_component.html.erb (hunk 2 above)
Comment (5andu) [reply app/components/time_component.html.erb:2]: I made it so ago appears, but it will be replaced by "auparavant" in French :)

**Comment by 5andu** [comment]
Done 👌

(I made "ago" appear, but I replaced it by "auparavant" in French)

Le sam. 15 oct. 2022 à 16:15, Joe Masilotti ***@***.***> a
écrit :

**Comment by joemasilotti** [comment]
Thanks so much for this!

**Pull request merged**